src/components/form/businessCreateProjectForm.tsx

```diff
@@ -114,86 +103,69 @@ interface projectDomain {
-  const [currDomains] = useState<any>([]);
+  const [domains, setDomains] = useState<Domain[]>([]);
 
-  const [projectDomains, setProjectDomains] = useState<any>([]); // add projectDomain
-  const [currProjectDomains, setCurrProjectDomains] = useState<any>([]);
-  const [tmpProjectDomains, setTmpProjectDomains] = useState<any>('');
+  const [projectDomains, setProjectDomains] = useState<projectDomain[]>([]);
+  const [currProjectDomains, setCurrProjectDomains] = useState<string[]>([]);
+  const [tmpProjectDomains, setTmpProjectDomains] = useState<string>('');
 
   const [loading, setLoading] = useState(false);
 
   const handleAddProjectDomain = () => {
-    if (
-      tmpProjectDomains &&
-      !currProjectDomains.some((domain: any) => domain === tmpProjectDomains)
-    ) {
+    if (tmpProjectDomains && !currProjectDomains.includes(tmpProjectDomains)) {
       setCurrProjectDomains([...currProjectDomains, tmpProjectDomains]);
       setTmpProjectDomains('');
     }
   };
 
   const handleDeleteProjectDomain = (domainToDelete: string) => {
     setCurrProjectDomains(
-      currProjectDomains.filter((domain: any) => domain !== domainToDelete),
+      currProjectDomains.filter((domain) => domain !== domainToDelete),
     );
   };
 
-  const handleAddSkill = () => {
-    if (tmpSkill && !currSkills.some((skill: any) => skill === tmpSkill)) {
-      setCurrSkills([...currSkills, tmpSkill]);
-      setTmpSkill('');
-    }
-  };
-
-  const handleDeleteSkill = (skillToDelete: string) => {
-    setCurrSkills(currSkills.filter((skill: any) => skill !== skillToDelete));
-  };
-
+  // Fetch data from APIs
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const projectDomainResponse =
-          await axiosInstance.get('/projectDomain/all');
-        console.log(
-          'projectDomain API Response get:',
-          projectDomainResponse.data.data,
-        );
-        const transformedProjectDomain = projectDomainResponse.data.data.map(
-          (skill: projectDomain) => ({
-            value: skill.label, // Set the value to label
-            label: skill.label, // Set the label to label
-          }),
-        );
-        setProjectDomains(transformedProjectDomain);
+        const [projectDomainResponse, domainResponse, skillsResponse] =
+          await Promise.all([
+            axiosInstance.get('/projectDomain/all'),
+            axiosInstance.get('/domain/all'),
+            axiosInstance.get('/skills/all'),
+          ]);
 
-        const domainResponse = await axiosInstance.get('/domain/all');
-        console.log('Domain API Response get:', domainResponse.data.data);
-        const transformedDomain = domainResponse.data.data.map(
-          (skill: Domain) => ({
-            value: skill.label, // Set the value to label
-            label: skill.label, // Set the label to label
-          }),
+        setDomains(
+          domainResponse.data.data.map((domain: any) => ({
```

Comment:
domainResponse.data.data.map this will throw fatal error if you don't get exact data in this api response. and will crash the app.
Add some type of validation check to see if the data you are going to set in state or map over exist at the expected location or not